src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -151,49 +151,50 @@ private void AddSource(string fileName, string source, bool isRootContextDef = f
                 string? @namespace = _currentContext.ContextType.Namespace;
                 bool isInGlobalNamespace = @namespace == JsonConstants.GlobalNamespaceValue;
 
-                StringBuilder sb = new(@"// <auto-generated/>
+                StringBuilder sb = new("""
+// <auto-generated/>
```

Comment:
This use of literals spanning newlines is going to build the newline of whatever platform this is compiled on into the source generator, so e.g. if the source generator is built on Windows, then when it's used on Linux it's going to emit \r\n for newlines.  Do we normalize that somewhere?

Reply:
Good question. We certainly don't do that on the source generator. Not sure if Roslyn does any kind of sanitization. If not it would seem that Linux is certainly tolerating carriage returns.

Reply:
> it would seem that Linux is certainly tolerating carriage returns.

The reverse would happen as well; if the source generator is built on Linux, then when used on Windows you'd end up with newlines being just \n.  I'm not sure how many tools are left that care about that, just pointing it out.